numpy/core/_add_newdocs.py

```diff
@@ -2322,7 +2322,7 @@
     See Also
     --------
     numpy.reshape : similar function
-    ndarray.reshape : similar method
+    numpy.ndarray.reshape : similar method
```

Comment:
The original was okay here, see: https://numpy.org/devdocs/reference/generated/numpy.ndarray.shape.html